test/e2e/testing-manifests/storage-csi/any-volume-datasource/setup-data-source-validator.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: volume-data-source-validator
+  namespace: kube-system
+spec:
+  serviceName: volume-data-source-validator
+  replicas: 1
+  selector:
+    matchLabels:
+      app: volume-data-source-validator
+  template:
+    metadata:
+      labels:
+        app: volume-data-source-validator
+    spec:
+      serviceAccount: volume-data-source-validator
+      containers:
+        - name: volume-data-source-validator
+          image: volume-data-source-validator:latest
```

Comment:
This should be a released image.

Reply:
I found that `k8s.gcr.io/sig-storage/volume-data-source-validator:v0.1.0` and `k8s.gcr.io/sig-storage/hello-populator:v0.1.0` are already published.  I've updated to use these images and opened issues to update the doc for these repo (and added references for the issues in the first comment).